_P208_Nokia_LCD_5110.ino

```diff
@@ -256,7 +262,7 @@ void setBacklight(struct EventStruct *event) {
   }
 }
 
-boolean displayText3(char deviceTemplate[][48], struct EventStruct *event ){ //48 = digits_per_line
+boolean displayText3(char deviceTemplate[][48], struct EventStruct *event ){ //14 must be digits_per_template_line
         String log = F("PCD8544: ");
         String string = F("");

```

Comment:
No need to initialize it with an empty string.
Default constructor of `String` is an empty string.

Reply:
Changed in next commit